### PR TITLE
Backend/upload image

### PR DESCRIPTION
**Pull request by eneshecan**
- Added uri field to tag model,
- tag/search_str endpoint returns also a uri now in json response,
- follow endpoint bug fixed,
- Comments added to endpoints to be visible in swagger.

### CODE REVIEW

File: lfc-api/LookingForConcerts/lfc_backend/views.py [hunk 1]
```diff
@@ -734,20 +738,22 @@ def get_tags(request, search_str):
         'format' : 'json',
         'language' : ['en', 'tr'],
         'search' : query,
-        'type':'item'
+        'type'   :'item'
```
Comment (berk94) [inline lfc-api/LookingForConcerts/lfc_backend/views.py]: Why is there an extra space here?

Comment (eneshecan) [reply lfc-api/LookingForConcerts/lfc_backend/views.py]: Thought alignment would be good for readability.

File: lfc-api/LookingForConcerts/lfc_backend/models.py [hunk 2]
```diff
@@ -58,7 +58,8 @@ class Tag(models.Model):
     '''
     tag_id = models.AutoField(primary_key=True)
     value = models.CharField(max_length=20)
-    context = models.CharField(max_length=20)
+    context = models.CharField(max_length=200, blank=True)
+    uri = models.CharField(max_length=50, blank=True)
```
Comment (berk94) [inline lfc-api/LookingForConcerts/lfc_backend/models.py]: Could you change 'uri' to 'wikidata_uri'?

File: lfc-api/LookingForConcerts/lfc_backend/views.py [hunk 3]
```diff
@@ -734,20 +738,22 @@ def get_tags(request, search_str):
...
     lenght =  len(json_response)
-
+    #print(json_response)
+    
     tags = []
     for i in range(lenght):
         if 'description' in json_response[i]:
             if any(re.findall(r'music|genre', json_response[i]['description'], re.IGNORECASE)):
-                value = json_response[i]['label']
+                value   = json_response[i]['label']
```
Comment (berk94) [inline lfc-api/LookingForConcerts/lfc_backend/views.py]: extra space?

File: lfc-api/LookingForConcerts/lfc_backend/views.py [hunk 4]
```diff
@@ -734,20 +738,22 @@ def get_tags(request, search_str):
...
                 context = json_response[i]['description']
-                t = '{"value":"'+value.replace('"','')+ '","context":"'+context.replace('"','')+'"}'
+                url     = json_response[i]['concepturi']
```
Comment (berk94) [inline lfc-api/LookingForConcerts/lfc_backend/views.py]: extra space?

File: lfc-api/LookingForConcerts/lfc_backend/views.py (hunk 1 above)
Comment (eneshecan) [reply lfc-api/LookingForConcerts/lfc_backend/views.py]: It is working like this by the way.

File: lfc-api/LookingForConcerts/lfc_backend/views.py (hunk 4 above)
Comment (eneshecan) [reply lfc-api/LookingForConcerts/lfc_backend/views.py]: This my convention of writing code. I try to align variables as much as possible.

File: lfc-api/LookingForConcerts/lfc_backend/views.py (hunk 3 above)
Comment (eneshecan) [reply lfc-api/LookingForConcerts/lfc_backend/views.py]: It is my styla.

**Pull request merged**